src/modules/m_filter.cpp

```diff
@@ -121,6 +126,8 @@ class FilterResult
 		std::string flags;
 		if (flag_no_opers)
 			flags.push_back('o');
+		if (flag_no_registered)
```

Comment:
This should be at the end because otherwise old versions will break when parsing flags.

Reply:
I've also modified helpop.conf.example and inspircd.conf.example, i hope can be good my english is so bad ...